packages/shopify-codemod/test/fixtures/assert/falsy-called-method-to-assert-not-called/basic.output.js

```diff
@@ -0,0 +1,2 @@
+assert.neverCalledWith(foo, bar, qux);
```

Comment:
Sorry if this is a silly comment once I have more context, but don't want to forget: will this properly deal with something like `assert.notOK(foo.calledTwice)`?